src/traits/circuit_supernova.rs

```diff
@@ -25,6 +27,15 @@ pub trait StepCircuit<F: PrimeField>: Send + Sync + Clone {
     pc: Option<&AllocatedNum<F>>,
     z: &[AllocatedNum<F>],
   ) -> Result<(Option<AllocatedNum<F>>, Vec<AllocatedNum<F>>), SynthesisError>;
+
+  /// AHHH
```

Comment:
This feels like it's halfway towards a confirmation of [Strehmel's results](https://cme.h-its.org/exelixis/pubs/JanThesis.pdf). Pray tell us more.